icons/bottle-dispenser.svg

```diff
@@ -0,0 +1,19 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="18.5" cy="5.5" r=".5" />
```

Comment:
This element should be avoided, see https://github.com/lucide-icons/lucide/pull/1436